docs/content/latest/quick-start/build-apps/go/ysql-pg-ssl.md

```diff
@@ -0,0 +1,335 @@
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PGX - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq-ssl/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pg-ssl/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PG - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [Go-pg client and ORM](https://pkg.go.dev/github.com/go-pg/pg), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register), to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.8](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+Refer to the [SSL/TLS configuration](../../../../secure/tls-encryption/client-to-server/) page to launch your YugabyteDB cluster with SSL/TLS enabled. This is already taken care of for you in **Yugabyte Cloud**, as the cluster is pre-configured to do client-side authentication with SSL/TLS enabled.
+
+### OpenSSL
+
+Install [OpenSSL](https://www.openssl.org/) 1.1.1 or later only if you have a YugabyteDB setup with SSL/TLS enabled. The **Yugabyte Cloud** clusters are always SSL/TLS enabled.
+
+### CA certificate
+
+Download the CA certificate file to allow your application connect securely to the YugabyteDB cluster. In case of a **Yugabyte Cloud** cluster,
+
+1. On the **Clusters** tab, select a cluster.
+
+1. Click **Connect**
+
+1. Click **Connect to your application** and download the CA cert.
+
+### Go PostgreSQL driver
+
+The [Go-pg/pg](https://github.com/go-pg/pg) is an ORM for Golang applications working with Postgresql. The current release of `pg v10` requires Go modules.
+
+To install the package locally, run the following commands:
+
+```sh
+$ mkdir yb-go-pg
+$ cd yb-go-pg
+$ go mod init hello
+$ go get github.com/go-pg/pg/v10
+```
+
+## Create the sample Go application
+
+Create a file `ybsql_hello_world.go` and copy the contents below.
+
+{{< note title="Note">}}
```

Comment:
Everywhere: This doesn't strike me as a note. Suggest the following, not in a note:

The `const` values are set to the defaults for a local installation of YugabyteDB. If you are using Yugabyte Cloud, replace the `const` values in the file as follows:
- `host` - The host address of your cluster. The host address is displayed on the cluster Settings tab.
- `user` - Your Yugabyte database username. In Yugabyte Cloud, the default user is `admin`.
- `password` - Your Yugabyte database password.
- `dbname` - The name of the Yugabyte database. The default Yugabyte database name is `yugabyte`.

`port` is set to 5433, which is the default port for the YSQL API.

Reply:
Clear structure , Absolutely using this. Thank you!